Client/Frontend/Settings/Main/Privacy/LoginsSetting.swift

```diff
@@ -44,6 +44,11 @@ class LoginsSetting: Setting {
     override func onClick(_: UINavigationController?) {
         deselectRow()
 
+        if CoordinatorFlagManager.isSettingsCoordinatorEnabled {
+            settings?.handle(route: .password)
```

Comment:
Oh I approved, but should we use delegate pattern here instead of calling the method on the VC directly?